baybe/utils/dataframe.py

```diff
@@ -418,3 +419,28 @@ def fuzzy_row_match(
             inds_matched.extend(inds_found)
 
     return pd.Index(inds_matched)
+
+
+def pretty_print_df(df: pd.DataFrame) -> str:
+    """Return a pretty/readable str object.
+
+    This function returns the first 10 rwos and 7 columns from the dataframe.
+    In case the dataframe is empty, it returns a corresponding statement.
+
+    Args:
+        df: The dataframe to be printed.
+
+    Returns:
+        The values to be printed as a str table.
+    """
+    # Handling with empty dataframe
+    if len(df) == 0:
```

Comment:
why do you make this distinction for length 0? What does the default str represnetation from pandas look like?

Reply:
The standard output from pandas gives a redundant output:
![image](https://github.com/emdgroup/baybe/assets/118910837/2fca504d-fea0-4666-8091-5284858de224)

Reply:
since a df could be empty but still have a set of columns assigned, Id prefer that default view. hence I see no need to have the custom `len(df)==0` check

Reply:
Okay this makes sence. I will remove the customized check 👍